src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -263,7 +263,7 @@ private function applyCurrentStyle(string $text, string $current, int $width, in
         }
 
         $lines = explode("\n", $text);
-        if ($width === $currentLineLength = \strlen(end($lines))) {
+        if ($width <= $currentLineLength += \strlen(end($lines))) {
```

Comment:
If I revert this line, tests still pass. Can you please add a test that covers the change (both of them: the `<=` and the `+=`)?

Reply:
I have upgrade the test and find a new problem inside the increment of `$currentLineLength`
fixed here:
https://github.com/symfony/symfony/pull/30852/files#diff-b79fd0c5df00b2bc5bf00f91b5939769R266